app/views/layouts/application.html.erb

```diff
@@ -176,7 +176,7 @@
       <% if current_user %>
         <% if current_user.editor? %>
           <br />
-          <h2><%= link_to t(:back_to_dashboard), url_for(controller: :admin, action: :index) %></h2>
```

Comment:
such notation did not worked for controllers in non-default namespace (e.g. Admin::FeaturedContentsController) so I've switched to url helpers.